unlock-app/package.json

```diff
@@ -38,31 +46,33 @@
     "eslint-plugin-react": "^7.7.0",
     "eslint-plugin-standard": "^4.0.0",
     "expect-enzyme": "^1.3.0",
+    "jest": "^23.6.0",
     "jest-enzyme": "^6.0.3",
     "nock": "^9.3.3",
+    "react-addons-test-utils": "15.6.2",
     "react-router-test-context": "^0.1.0",
+    "react-test-renderer": "16.4.2",
     "redux-mock-store": "^1.5.1",
-    "storybook-react-router": "^1.0.1"
+    "storybook-react-router": "^1.0.1",
+    "webpack": "^4.20.2"
   },
   "engines": {
     "node": "=8.11.4"
   },
   "engineStrict": true,
   "scripts": {
-    "start-app": "react-scripts start",
+    "dev": "npm run prestart && next src",
```

Comment:
For dev environment, explicitly running `npm run prestart` with `npm run dev`.  `prestart` will automatically run in production.